### PR TITLE
contribuiting guidelines and lint command

### PR DESCRIPTION
**Pull request by Ryan-McBride**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue.
```
Comment (tristandenyer) [inline CONTRIBUTING.md]: Can you link to Issues like you did with code of conduct?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Development Process
+
+We follow [Git Flow](https://guides.github.com/introduction/flow/) for changes to this repository.
+
+  1. Fork this repo to create a copy hosted on your github account. The Fork button is in the top right of the page.
+  1. Clone down your copy of this repo onto your local machine: `git clone <YOUR GITHUB REPO URL>`
+  1. Navigate to the new directory git created. `cd postman-docs`
+  1. Checkout the `develop` branch. `git checkout develop`
+  1. Create a new branch for your work based on develop `git checkout -b <YOUR BRANCH NAME>` Your branch name should be something descriptive of the changes you wish to make
```
Comment (tristandenyer) [inline CONTRIBUTING.md]: append line with: ", and include the Issue number this change is associated with. Example: `1234-update-mock-api-docs`."

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  1. Run `npm install` to add all dependencies
```
Comment (tristandenyer) [inline CONTRIBUTING.md]: Do we need to do npm install? Esp since we are not talking about a local build?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  1. Make your changes. The Learning Centers documentation is built from the `.md` files in the `v6` folder of the repo
```
Comment (tristandenyer) [inline CONTRIBUTING.md]: Change "Learning Centers" to "Learning Center's"

File: CONTRIBUTING.md (hunk 1 above)
Comment (Ryan-McBride) [reply CONTRIBUTING.md]: added

File: CONTRIBUTING.md (hunk 3 above)
Comment (Ryan-McBride) [reply CONTRIBUTING.md]: It's required to run the markdown linter locally, which is currently a required step

File: CONTRIBUTING.md (hunk 4 above)
Comment (Ryan-McBride) [reply CONTRIBUTING.md]: fixed 👍

File: CONTRIBUTING.md (hunk 3 above)
Comment (tristandenyer) [reply CONTRIBUTING.md]: oh, got it

**Pull request merged**